### ISSUE TITLE
Awful setup process

### ISSUE THREAD

**Issue by patrickswynne** [MicrosoftDocs/xamarin-docs#2103]
5 hours in and still this isn't working. I think this has been probably the worst setup experience I have ever had. If it was up to me I would get a new time on fixing this. 5 year old product that still doesn't come ready out of the box, absolute garbage.

**Comment by conceptdev** [comment]
Thanks for your feedback. If you wish to submit an issue [via Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019) the additional information it provides could help us improve.

**Comment by patrickswynne** [comment]
I get an error about missing qt_setup whenever I try to run any emulator. I have tried many different versions of android and screen resolutions. I have attempted to install this on 2 different windows machines and tried to install it on a mac that didn't work for another reason to do with xcode versions.

ERROR: qt_setup.cpp:28: Qt library not found at C:\Program Files (x86)\Android\android-sdk\emulator\lib64\qt\lib
C:\Program Files (x86)\Android\android-sdk\emulator\qemu\windows-x86_64\qemu-system-x86_64.exe: -no-boot-anim: invalid option
Emulator em1 cannot be started.
